bindgen/ir/var.rs

```diff
@@ -168,14 +168,33 @@ fn handle_function_macro(
     };
     let tokens: Vec<_> = cursor.tokens().iter().collect();
     if let Some(boundary) = tokens.iter().position(is_closing_paren) {
-        let mut spelled = tokens.iter().map(ClangToken::spelling);
-        // Add 1, to convert index to length.
-        let left = spelled.by_ref().take(boundary + 1);
-        let left = left.collect::<Vec<_>>().concat();
-        if let Ok(left) = String::from_utf8(left) {
-            let right: Vec<_> = spelled.collect();
-            callbacks.func_macro(&left, &right);
-        }
+        let mut tokens = tokens
+            .iter()
+            .map(|token| {
+                let s = token.spelling();
+                encoding_rs::mem::decode_latin1(s)
```

Comment:
This is needed for `bindgen-integration/cpp/Test.h`, which is encoded in Latin-1. Not sure if that is on purpose, but doesn't hurt to support both UTF-8 and Latin-1.

Reply:
Seems like this is wrong, since Latin-1 is not a subset of UTF-8.